images.txt

```diff
@@ -11,14 +11,14 @@ bitnami/minio-client:latest
 bitnami/prometheus:latest
 bitnami/thanos:latest
 busybox:latest
-cockroachdb/bazel:latest
+cockroachdb/bazel:latest-do-not-use
```

Comment:
??

Reply:
Yeah, that was weird. I used it because `rumble` doesn't actually need to "use" the image and that was the only `latest`-like tag available. But I am just going to drop that :) Good call.